README.md

```diff
@@ -65,7 +65,7 @@ The `options` parameter is optional and allows for clients to override the defau
 
 ```javascript
 const patientClient = new plaid.Client(client_id, secret, public_key, plaid_env, {
-  timeout: 10 * 60 * 1000, // 30 minutes
+  timeout: 10 * 60 * 1000, // 10 minutes
```

Comment:
```suggestion
  timeout: 30 * 60 * 1000, // 30 minutes
```
Because the default timeout is actually 10min (and this is the "patient" example), I believe we actually want the specified timeout to be 30min.